RevenueCat/Scripts/Purchases.cs

```diff
@@ -260,13 +285,34 @@ public void SyncPurchases()
         _wrapper.SyncPurchases();
     }
 
+    /// <summary>
+    /// Android only. Noop in iOS.
+    /// 
+    /// This method will send a purchase to the RevenueCat backend. This function should only be called if you are
+    /// in Amazon observer mode or performing a client side migration of your current users to RevenueCat.
+    /// The receipt IDs are cached if successfully posted so they are not posted more than once.
+    /// </summary>
+    /// <param name="productID">Product ID associated to the purchase.</param>
+    /// <param name="receiptID"> ReceiptId that represents the Amazon purchase.</param>
+    /// <param name="amazonUserID">Amazon's userID.</param>
+    /// <param name="isoCurrencyCode">Product's currency code in ISO 4217 format.</param>
+    /// <param name="price">Product's price.</param>
+    public void SyncObserverModeAmazonPurchase(string productID, string receiptID, string amazonUserID,
```

Comment:
Hmm thinking out loud here. Do we wait to have a version compatible with Unity IAP to release this function? Just saying because this version won't be compatible with Unity IAP due to incompatibilities with the BillingClient version. Although if someone is targeting the Amazon Store exclusively, it will work since the BillingClient won't be used 🤔